.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -190,26 +191,30 @@ jobs:
       run: rustup component add clippy rustfmt
 
     - name: "Rust tools cache"
-      uses: actions/cache@v3
+      uses: actions/cache@v4
       id: rust-tools-cache
       with:
         path: |
           ${{ matrix.cargo_dir }}/bin/sccache${{ matrix.exe_suffix }}
           ${{ matrix.cargo_dir }}/bin/mdbook${{ matrix.exe_suffix }}
           ${{ matrix.cargo_dir }}/bin/mdbook-linkcheck${{ matrix.exe_suffix }}
-        key: ${{ matrix.os }}_sccache-0.4.1_mdbook-0.4.28_mdbook-linkcheck-0.7.7
+        key: ${{ matrix.os }}_sccache-0.7.6_mdbook-0.4.36_mdbook-linkcheck-0.7.7
     - name: "Build Rust tools"
       if: steps.rust-tools-cache.outputs.cache-hit != 'true'
-      run: cargo install --locked sccache mdbook mdbook-linkcheck
+      # Do not build with storage backends enabled, we only need local
+      run: |
+        cargo install --no-default-features sccache
```

Comment:
there is a GitHub cache storage backend, wonder if this is better or worse than using local ?

https://github.com/mozilla/sccache/blob/main/docs/GHA.md

Reply:
But looks like that would download on the fly, so wonder if that is slower than downloading a compressed local blob then querying that

Reply:
I think this is something we can just try out in a different PR.